### ISSUE TITLE
Block Editor: Tooltips for the icons in the block toolbar obscure the interface

### ISSUE THREAD

**Issue by gziolo** [WordPress/gutenberg#23826]
**Describe the bug**
Tooltips for the icons in the block toolbar obscure the interface. They should be positioned consistently in a way that allows seeing all items in the toolbar.

**To reproduce**
1. Open a post with multiple blocks.
2. Move focus to the block toolbar for Heading block.
3. Navigate with arrows and observer where tooltips render.

**Expected behavior**

All items in the toolbar and dropdown menus should be visible while rendering tooltips.

**Screenshots**
![tooltipts-bottom](https://user-images.githubusercontent.com/699132/87035696-ff1f3f00-c1e9-11ea-97e4-0b7da3ce1297.gif)

**Editor version (please complete the following information):**
- WordPress version: WordPress 5.5.0-beta.1
- Does the website has Gutenberg plugin installed, or is it using the block editor that comes by default? Gutenberg plugin build from the latest `master` branch

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Safari
 - Version 13.1.1

**Comment by noisysocks** [comment]
Where should the H1 H2 H3 etc. buttons appear? Above the toolbar? Or should the entire toolbar move up?

**Comment by richtabor** [comment]
> Where should the H1 H2 H3 etc. buttons appear? Above the toolbar? Or should the entire toolbar move up?

The H1 H2 H3 etc button are fine - its the tooltips that we're reviewing (how they're covering up other UI elements). :)

**Comment by youknowriad** [comment]
This seems to work properly for me, I can't seem to reproduce the bug in the gif?

**Comment by shaunandrews** [comment]
I don't understand the problem here. Are you suggesting that tooltips render somewhere other than next to the element they reference?

**Comment by youknowriad** [comment]
I'm going to close this issus as it's not clear what the bug is right now. Let's reopen if we have more details.

**Comment by afercia** [comment]
What I see in the GIF and can also reproduce on 5.5 Beta 3 is:

1
- when pressing the "Change heading level" button, the button tooltip initially stays in position
- then focus is moved to the first heading level in the popup toolbar
- at this point, the previous tooltip disappears and a new tooltip appears for the focused button
- this creates a not pleasant effect of "movement", as the tooltip was moving on the screen

2
The new tooltip design, without the "arrow" makes the tooltip partially hide most (any?) of the various UIs it's used for. I'm not sure I understand the reasons for this design change and not sure it's a improvement.

Example in WordPress 5.5:

<img width="414" alt="Screenshot 2020-07-24 at 14 06 38" src="https://user-images.githubusercontent.com/1682452/88390217-39770780-cdb8-11ea-976a-e1d546c49c1b.png">


Example in WordPress 5.4:

<img width="425" alt="Screenshot 2020-07-24 at 14 08 30" src="https://user-images.githubusercontent.com/1682452/88390232-4136ac00-cdb8-11ea-8cec-f6a605ec611b.png">

3
The tooltips don't expose the keyboard shortcuts any longer. This was a welcomed accessibility feature as it greatly improved discoverability of the shortcuts. Now they're gone, not sure what is the reasoning behind this change and not sure it's an improvement.

**Comment by gziolo** [comment]
The most visible issue is with `Move up` icon that is displayed over `Move down`. If you still don't think it isn't an issue we can close this one as it seems to be a minor thing.
